types/node-fetch/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
     "dependencies": {
-        "form-data": "^2.3.3"
+        "form-data": "2.5.0"
```

Comment:
Why not `^`?

Reply:
(I mean other than the fact that I didn't write it in my suggestion :) )

Reply:
amended